dojo/settings/settings.dist.py

```diff
@@ -931,7 +931,7 @@ def saml2_attrib_map_format(dict):
         'xmlsec_binary': '/usr/bin/xmlsec1',
 
         # your entity id, usually your subdomain plus the url to the metadata view
-        'entityid': '%s' % SAML2_ENTITY_ID,
+        'entityid': f'{SAML2_ENTITY_ID}',
```

Comment:
Not sure if the `str()` is even really necessary here, but no need for a format string

```suggestion
        'entityid': str(SAML2_ENTITY_ID),
```